specifyweb/frontend/js_src/lib/components/FormPlugins/LatLongUi.tsx

```diff
@@ -74,6 +75,11 @@ function Coordinate({
     [resource, coordinateField, updateValue, step, fieldType]
   );
 
+  const [_blockers, setBlockers] = useSaveBlockers(
+    resource,
+    resource.specifyTable.fields.find((field) => field.name === coordinateField)
```

Comment:
don't do .fields.find() - use the specifyTable.strictGetField() util instead

and, on line 38 the strictGetField is already called - extract that usage into a variable and reuse it here